ci/build-docker.sh

```diff
@@ -16,36 +16,77 @@
 
 set -eu
 
+if [ "${TERM:-}" = "dumb" ]; then
+  readonly COLOR_RED=""
+  readonly COLOR_GREEN=""
+  readonly COLOR_YELLOW=""
+  readonly COLOR_RESET=""
+else
+  readonly COLOR_RED=$(tput setaf 1)
+  readonly COLOR_GREEN=$(tput setaf 2)
+  readonly COLOR_YELLOW=$(tput setaf 3)
+  readonly COLOR_RESET=$(tput sgr0)
+fi
+
+# This script is supposed to run inside a Docker container, see
+# ci/build-linux.sh for the expected setup.  The /v directory is a volume
+# pointing to a (clean-ish) checkout of google-cloud-cpp:
+(cd /v ; ./ci/check-style.sh)
+
 # Run the configure / compile / test cycle inside a docker image.
 # This script is designed to work in the context created by the
 # ci/Dockerfile.* build scripts.
-mkdir -p gccpp/build-output
-cd gccpp/build-output
+readonly IMAGE="cached-${DISTRO}-${DISTRO_VERSION}"
+mkdir -p build-output/${IMAGE}
+cd build-output/${IMAGE}
 
 CMAKE_COMMAND="cmake"
 if [ "${SCAN_BUILD}" = "yes" ]; then
   CMAKE_COMMAND="scan-build cmake"
 fi
-${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" ${CMAKE_FLAGS:-} ..
 
-# If scan-build is enabled we, need to manually compile the dependencies;
+echo "travis_fold:start:configure-cmake"
```

Comment:
Does Travis actually understand that as a directive and this is how it adds those collapsible sections?

Reply:
Yes, for example see:

https://travis-ci.org/coryan/google-cloud-cpp/jobs/320346069

It is sort of documented or officially not documented in travis-ci/travis-ci#2285